components/button/metadata/button-pending.yml

```diff
@@ -0,0 +1,328 @@
+name: Button - pending
+status: Verified
+description: The pending button is for indicating that a quick progress action is taking place. In this case, the label and optional icon disappear and a progress circle appears. The progress circle always shows an indeterminate progress. We recommend the use of the `.is-pending` class on the component's parent container, but there is also an option to use an attribute of `pending` instead. Buttons should have the `disabled` attribute when the pending state is applied.
+SpectrumSiteSlug: https://spectrum.adobe.com/page/button/#Pending
+examples:
+  - id: button-pending
+    name: Pending for accent, negative, primary, and secondary buttons
```

Comment:
Would it be worth showing examples for pending on buttons that are accent, negative, primary, or secondary? It looks like these are not any of those variants and are showing on a default button without any variant class (that is looking like 'spectrum-Button--outline'---is button even supposed to be used without any variant set?). 

In Storybook, I can see that with accent, it will have a grey disabled background that I don't see on this page. You could show both outline and accent, since there only appears to be two different visual styles?